torch/_subclasses/fake_tensor.py

```diff
@@ -470,6 +470,8 @@ def __new__(cls, fake_mode, elem, device, constant=None):
             dispatch_device=True,
             device_for_backend_keys=device,
         )
+        if not fake_mode._allow_unsafe_data_ptr_access:
+            torch._C._set_throw_on_mutable_data_ptr(self)
```

Comment:
Why only on mutable data_ptr and not all access?

Reply:
It's mostly that I'm minimizing the blast radius